quickwit/rest-api-tests/run_tests.py

```diff
@@ -100,7 +100,10 @@ def run_request_step(method, step, previous_result):
     body_from_file = step.get("body_from_file", None)
     if body_from_file is not None:
         body_from_file = osp.join(step["cwd"], body_from_file)
-        kvargs["data"] = load_data(body_from_file)
+        kvargs["data"] = open(body_from_file, 'rb').read()
+        if body_from_file.endswith("gz"):
+            kvargs.setdefault("headers")["content-encoding"] = "gzip"
```

Comment:
can we add a test with a gzip body?

Reply:
The test already has gzip data (`gharchive-bulk.json.gz`), I just deactivated the client side decompression